recipes-debian/bluez5/bluez5_debian.bbappend

```diff
@@ -0,0 +1,5 @@
+FILESEXTRAPATHS_prepend := "${THISDIR}/bluez5:"
+
+SRC_URI += " \
+	file://tools_Fix_build_after_y2038_changes_in_glibc.patch \	
+"
```

Comment:
Please include this into bluez5_debian.bb if the patch has no side-effect for kernel 4.19.

Reply:
@hiraku-wfs I forgot to close this PR. This patch is merged in [this PR](https://github.com/miraclelinux/meta-debian-extended/pull/306)

Reply:
I also forgot it. Thanks.